packages/framework/src/Model/Product/Elasticsearch/ProductExportCronModule.php

```diff
@@ -4,25 +4,33 @@
 
 namespace Shopsys\FrameworkBundle\Model\Product\Elasticsearch;
 
-use Shopsys\FrameworkBundle\Component\Domain\Domain;
-use Shopsys\FrameworkBundle\Component\Elasticsearch\AbstractExportCronModule;
-use Shopsys\FrameworkBundle\Component\Elasticsearch\IndexDefinitionLoader;
-use Shopsys\FrameworkBundle\Component\Elasticsearch\IndexFacade;
+use Shopsys\FrameworkBundle\Model\Product\Recalculation\ProductRecalculationDispatcher;
+use Shopsys\Plugin\Cron\SimpleCronModuleInterface;
+use Symfony\Bridge\Monolog\Logger;
 
-class ProductExportCronModule extends AbstractExportCronModule
+class ProductExportCronModule implements SimpleCronModuleInterface
```

Comment:
You nicely summarized my thoughts exactly 😄 
I would like to keep the cron module (at least for now). It may be easily disabled in a project if it becomes unnecessary. 

Good thinking about the renaming though. Done